### DISCUSSION TITLE
If AliPay were an option

### DISCUSSION THREAD

**Discussion by Rangtian** [Tampermonkey/tampermonkey#1661] [General]
I have neither PayPal nor credit card.

![image](https://user-images.githubusercontent.com/22640407/204435450-00bd8174-b1da-491b-b6da-6e3d7d4c8568.png)

**Comment by derjanb** [comment]
Please see https://github.com/Tampermonkey/tampermonkey/issues/674#issuecomment-670414131

**Reply by caoyuanlang** [reply]
能不能加上微信和支付宝的那
